pkg/storkctl/schedulepolicy_test.go

```diff
@@ -212,21 +212,32 @@ func TestCreateSchedulePolicyFailureCases(t *testing.T) {
 	testCommon(t, cmdArgs, nil, expected, true)
 
 	//Invalid Day value for Weekly Policy
-	cmdArgs = []string{"create", "schedulepolicy", "test-policy", "-t", "Weekly", "--time", "12:05PM", "--day-of-week", "SUNDAY"}
-	expected = "error: Invalid day of the week (SUNDAY) in Weekly policy"
+	cmdArgs = []string{"create", "schedulepolicy", "test-policy", "-t", "Weekly", "--time", "12:05PM", "--day-of-week", "FUNDAY"}
+	expected = "error: Invalid day of the week (funday) in Weekly policy"
 	testCommon(t, cmdArgs, nil, expected, true)
 
 	//Invalid Date value for Monthly Policy
 	cmdArgs = []string{"create", "schedulepolicy", "test-policy", "-t", "Monthly", "--date-of-month", "32"}
 	expected = "error: Invalid date of the month (32) in Monthly policy"
 	testCommon(t, cmdArgs, nil, expected, true)
+
+	//Invalid retain value
+	cmdArgs = []string{"create", "schedulepolicy", "test-policy", "--retain", "0"}
+	expected = "error: need to provide a valid value for retain. It should be a positive integer"
```

Comment:
nit: Need.

Reply:
Addressed in the revision

Reply:
I looked at the storkctl codebase. Error messages most of the places starts with a lowercase letter only.
Also I tried changing it to Invalid, however then travis build fails due to staticcheck

pkg/storkctl/schedulepolicy.go:119:20: error strings should not be capitalized (ST1005)
Makefile:85: recipe for target 'staticcheck' failed
make: *** [staticcheck] Error 1